MANIFEST.in

```diff
@@ -8,4 +9,5 @@ recursive-include tests *
 recursive-include docs *
 prune docs/_build
 recursive-exclude * __pycache__
-recursive-exclude * *.pyc
\ No newline at end of file
+recursive-exclude * *.pycinclude
```

Comment:
Was the L12 edit intentional?

Reply:
I think this is a mistake. Thanks for spotting it.